ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/workers-rhel8/provision/cucushift-installer-rehearse-ibmcloud-ipi-workers-rhel8-provision-chain.yaml

```diff
@@ -0,0 +1,26 @@
+chain:
+  as: cucushift-installer-rehearse-ibmcloud-ipi-workers-rhel8-provision
+  steps:
+  - ref: ibmcloud-provision-vpc
+  - chain: ibmcloud-provision-bastionhost
+  - chain: ipi-conf-ibmcloud
+  - ref: ipi-conf-ibmcloud-provisionednetwork
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  - ref: workers-rhel-ibmcloud-provision
```

Comment:
Can you confirm how many size the rhel's root partition has?

Reply:
RHEL_VM_DISK_SIZE set to 100, and this is the minimized value for boot-volume

>  volumeJson=$(jq -n \
    --arg vn "$volume" \
    --argjson size ${RHEL_VM_DISK_SIZE} \
    '{"name": $vn, "volume": {"capacity": $size, "profile": {"name": "general-purpose"}}}')
  cmd="${IBMCLOUD_CLI} is instance-create $vmName ${vpcName} ${zone} ${RHEL_VM_SIZE} ${subnet} --image ${RHEL_IMAGE} --keys $preDefaultKey --sgs ${sg},${sg1} --boot-volume '${volumeJson}'"